tools/python/boutiques/localExec.py

```diff
@@ -413,7 +413,7 @@ def prepare(self):
 
             if self.imagePath:
                 conName = op.basename(self.imagePath)
-                imageDir = op.dirname(self.imagePath)
+                imageDir = op.dirname(self.imagePath) or "./"
```

Comment:
Not sure if this is portable: replace with `os.getcwd()`?

Reply:
The dirname of a plain basename is `.`, so it's fine.

A comment though. I'd not use the trailing `/`, that's bad semantics. Accessing directories with a trailing slash has different behavior on different unix system. The name of the dir is really `.`.

Reply:
But on Windows this may not work, right @prioux ? We haven't yet adding testing to Windows, but that is the goal, so `os.getcwd()` would have a higher likelihood of being robust to the switch, I think?

Reply:
I think windows support "." to mean the current directory... otherwise how do you even specify the current working directory?

Reply:
That being said, I'm fine with this PR.

Reply:
There's already #398 for windows compatibility anyway. And lots of stuff in here will break way before all the path operations are adjusted (e.g. popen())

Reply:
I fond something that will please everyone!

`os.path.normpath("")` returns "." and will probably work OK on windows too (whatever the convention over there is).

So the code above could be written as

```python
imageDir = op.normpath(op.dirname(self.imagePath))
```

Reply:
(Tested in interactive python 2.7.5)